ClosedXML/Excel/XLWorkbook.cs

```diff
@@ -857,11 +852,41 @@ public void AddWorksheet(IXLWorksheet worksheet)
             worksheet.CopyTo(this, worksheet.Name);
         }
 
+        /// <summary>
+        /// Add a worksheet with a table at Cell(row:1, column:1). The datatable's name is used for the
+        /// worksheet name and the table name.
+        /// </summary>
+        /// <param name="dataTable">Datatable to insert</param>
+        /// <returns>Inserted Worksheet</returns>
+        public IXLWorksheet AddWorksheet(DataTable dataTable)
```

Comment:
Please place comments to the interface, not an implementation. That way, users will see them through IntelliSense.

Reply:
done